src/styles/index.scss

```diff
@@ -1,3 +1,67 @@
+@import './variables';
+
 body {
   margin: 0;
+  padding: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  width: 726px;
+  box-sizing: border-box;
+  padding: $padding;
+
+  &__space {
+    display: flex;
+    flex-wrap: wrap;
+    gap: 1px;
+  }
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: $square-size;
+    width: $square-size;
+    box-sizing: border-box;
+    border: $border-base;
+    background-color: $square-color;
+
+    &:hover {
+      cursor: pointer;
+      animation-name: move;
+      animation-duration: 0.5s;
+    }
+  }
+}
+
+@for $number from 1 through 31 {
+  .calendar__day--#{$number}::before {
+    content: '#{$number}';
+    font-family: $font-family;
+    font-size: $font-size;
+  }
+}
+
+@each $name, $day in $days-of-week {
+  .calendar__startDay--#{$name} .calendar__day--1 {
+    margin-left: calc($day * 100 + $day);
```

Comment:
And if we want to make another `gap`? For example - 3?
Create a variable for ur `gap` and add it to the formula